demo/demo.js

```diff
@@ -0,0 +1,34 @@
+const demos = [
```

Comment:
This complicates the demos a bit.  It's easier for a consumer to just see a single html file that has everything in it.  A lot of Cytoscape users are junior programmers and scientists (novice programmers), so it's best to keep things as simple as possible.

Reply:
The demo shell is simpler (and doesn't repeat itself for all of the dependencies and CSS files) and each of the demo JS files is now self-contained. I think it's a good tradeoff... In the longer run, a demo where you can toggle options interactively and see the resultant `cy` incantation would be even cooler.

Reply:
I agree that it may be simpler for maintainers to have a more complex demo, but it's not simpler for consumers.  Consumers are a higher priority.

Reply:
I still believe this is simpler for consumers too, since they don't need to read through the boilerplate HTML. The demo JS files are (aside from the larger demo-demo) basically single Cytoscape calls.

Reply:
Fair enough.  We have used the individual file pattern for demos in all of the Cytoscape extensions, and we haven't gotten any complaints from users.

Would you commit to providing support for these demos if users end up finding them harder to understand than before?

Reply:
Sure, you can mention me in issues/... if folks have issues with this :)